dot/telemetry/mailer_test.go

```diff
@@ -106,13 +119,13 @@ func TestHandler_SendMulti(t *testing.T) {
 			"1"),
 	}
 
-	resultCh = make(chan []byte)
-
+	resultCh := make(chan []byte)
+	mailer := bootstrapMailer2Test(t, resultCh)
```

Comment:
Maybe you could have the assertion code inside the handler instead of piping the handler data into the resultsCh?

Reply:
yes, I will need to pass as argument an `http.HandlerFunc` instead of a channel and for each test write its own handler, it's a good suggestion, I will try to implement